table/block.h

```diff
@@ -426,14 +375,14 @@ class DataBlockIter final : public BlockIter {
                global_seqno, read_amp_bitmap, block_contents_pinned);
   }
   void Initialize(const Comparator* comparator,
-                  const Comparator* user_comparator, const char* data,
+                  const Comparator* /*user_comparator*/, const char* data,
```

Comment:
Are you leaving the removal of user_comparator from this function to future PRs?